core/examples/luigi-sample-vue/README.md

```diff
@@ -0,0 +1,51 @@
+# Sample luigi application written in Vue.JS
+
+## Development
+
+This mono repository uses [Lerna](https://lernajs.io/) for project management. Ensure that you have installed lerna, if not then run   
+`npm install --global lerna`
+
+To start developing this application, follow these steps:
+
+1. Install dependencies.  
+`lerna bootstrap` executes npm install and links cross-dependencies
+```bash
+lerna bootstrap
+```
+
+2. Bundle cross-dependencies.  
+`lerna run` runs the given script in every package referenced in lerna.json
+```bash
+lerna run bundle
+```
+
+## Run server
+
+### Compiles and hot-reloads for development
```

Comment:
Too many headings. Maybe you could make it a numbered list (1,2,3...) if this is a step-by-step instruction, or simply bullet points?

Also: please make it more descriptive.